percy/graphql-mocks.js

```diff
@@ -0,0 +1,73 @@
+const PlanResponseBike = require('./mocks/PlanResponseBike.json').data.plan
+const PlanResponseWalk = require('./mocks/PlanResponseWalk.json').data.plan
+const PlanResponseBusSubwayTram =
+  require('./mocks/PlanResponseBusSubwayTram.json').data.plan
+const TripResponse = require('./mocks/TripResponse.json').data.trip
+const NearestResponse = require('./mocks/NearbyResponse.json').data.nearest
+const Stop114900Response = require('./mocks/Stop114900Response.json').data.stop
+const Stop803Response = require('./mocks/Stop803Response.json').data.stop
+const StopsByRadiusResponse = require('./mocks/StopsByRadiusResponse.json').data
+  .stopsByRadius
+const ServiceTimeRangeResponse =
+  require('./mocks/ServiceTimeRangeResponse.json').data.serviceTimeRange
+const RoutesResponse = require('./mocks/Routes.json').data.routes
+const IndividualRouteResponse = require('./mocks/IndividualRoute.json').data
+  .route
+
+function getPlanResponseMock(transportModes) {
+  const transportModesString = transportModes
+    .map((tm) => tm.mode)
+    .sort()
+    .join('')
+  switch (transportModesString) {
+    case 'BICYCLEBUSSUBWAYTRAM':
+    case 'BUSSUBWAY':
+      return PlanResponseBusSubwayTram
+    case 'BICYCLE':
+      return PlanResponseBike
+    case 'WALK':
+      return PlanResponseWalk
+    default:
+      return PlanResponseBike
+  }
+}
+
+function getStopResponseMock(stopId) {
+  switch (stopId) {
+    case 'MARTA:803':
+      return Stop803Response
+    case 'MARTA:114900':
+    default:
+      return Stop114900Response
+  }
+}
+
+const mocks = {
```

Comment:
I could write something to do that, it isn't a bad idea. I could put some assertions in the code for the number of times we expect each resolver to be called.

Reply:
Actually I can't do this very easily, the assertions won't work in the mocking server since it's run as a subprocess under the Percy tests. For some reason importing the server and running it directly in the Percy test doesn't work very well, it fails part way through. 
I tried adding some code that logs the calls to each resolver at the end, but the function doesn't seem to be called. 

Open to ideas here.